app/controllers/cart_items_controller.rb

```diff
@@ -13,7 +14,7 @@ def create
   end
   
   def update
-    current_cart.delete_item(params['item']['beer_id'])
+    @cart.delete_item(params['item']['beer_id'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.